kex

```diff
@@ -43,7 +45,10 @@ EOF
 }
 
 po_list() {
-  kubectl get pods -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}'
+  local NS_PARAM="--namespace=${NAMESPACE:-}"
```

Comment:
@metadave @lypht This is not an order issue. The problem is our `getopts` cases assumes exit 0 immediately for l and h… with the idea neither should be combined. So because of that, yes there is an order now between the two. I'm going to propose a simple fix in a moment.